src/components/VideosList/VideosList.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import VideosList from './VideosList.component';
+import data from '../../mocks/youtube-videos-mock.json';
+
+describe('VideosList', () => {
+  it('exists', () => {
+    render(React.createElement(VideosList, { videos: [] }));
+  });
+
+  it('renders message when there are no videos', () => {
+    render(React.createElement(VideosList, { videos: [] }));
```

Comment:
Instead of doing it this way, it is better to render the actual component with:

```javascript
render(<VideosList videos={[]} />);
```

Reply:
The linter gave me an error because you can not have jsx outside of jsx components.
what should I do?

Reply:
Since you will be using JSX inside the test files you should change the file extension to `.jsx` as well. That should also fix the linting error.

Reply:
Thanks a lot!! :)